### PR TITLE
added the source code

### PR DESCRIPTION
**Pull request by igorkotof**
Added source code of the module

### CODE REVIEW

**Comment by igorkotof** [comment]
[Create terraform-aws-route53-healthcheck opensource module](https://trello.com/c/CDoXo67N/124-create-terraform-aws-route53-healthcheck-opensource-module)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+The module creates: 
+* SNS Topic
+* SNS Subscription (e.g. PagerDuty)
+* Route53 Healthcheck
+* Cloudwatch Metric Alarm
+* (optional) Secondary Route53 record for failover routing policy to public S3 website with "Maintenance page"
+
+### Example:
+We will monitore s3-static-website.s3-website-us-east-1.amazonaws.com with PagetDuty notification with
```
Comment (DimmKirr) [inline README.md]: `monitor` typo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
 # terraform-aws-route53-healthcheck
 Terraform AWS Route53 Healthcheck Module
+
+The module creates: 
+* SNS Topic
+* SNS Subscription (e.g. PagerDuty)
+* Route53 Healthcheck
+* Cloudwatch Metric Alarm
+* (optional) Secondary Route53 record for failover routing policy to public S3 website with "Maintenance page"
```
Comment (DimmKirr) [inline README.md:9]: Let's add information about s3 bucket?

File: versions.tf [hunk 3]
```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+  required_version = ">= 0.13"
```
Comment (DimmKirr) [inline versions.tf]: Can we make this compatible with 0.12 (even if with warnings)?

**Review by DimmKirr — CHANGES_REQUESTED** [review]

**Comment by igorkotof** [comment]
@AutomationD Resolved all comments - Thanks!

**Comment by igorkotof** [comment]
@AutomationD fixed  :) sorry

**Pull request merged**